nemo_skills/inference/check_contamination.py

```diff
@@ -143,11 +137,12 @@ def check_contamination(cfg: CheckContaminationConfig):
                                     f'{cfg.retrieve_key}1': similar_item,
                                 }
                             )
+
+                prompts = [prompt.fill_prompt(dp) for dp in all_data]
+
                 if cfg.prompt_template:
-                    prompts = [prompt.build_string(dp) for dp in all_data]
                     stop_phrases = list(prompt.config.template.stop_phrases)
```

Comment:
Let's create a function for this as well in the prompt - get_stop_phrases or something like that? Returns from template or none, so that we can fully remove the ifs

Reply:
I like it, will do